src/components/Layout/Toolbar.css

```diff
@@ -20,9 +20,3 @@
 .Logo {
   height: 80%;
 }
-
-@media (max-width: 499px) {
```

Comment:
Is there a common set of min/max-width values for most form factors or are people just whacking in random numbers like this?